tests/_internal/concurrency/test_api.py

```diff
@@ -304,34 +304,6 @@ def test_from_sync_call_in_loop_thread(work):
     wait_for_global_loop_exit()
 
 
-@pytest.mark.parametrize("work", [identity, aidentity])
```

Comment:
these were testing the static method from `base` that was never actually used in the library, which was also inlined into the `call_soon_in_waiting_thread` methods from `from_async` and `from_sync` in this PR